documentation/modules/configuring/ref-storage-ephemeral.adoc

```diff
@@ -32,9 +44,9 @@ spec:
     # ...
 ----
 
-== Log directories
+== Ephemeral storage for log directories
```

Comment:
I think this title is a bit confusing. I would definitely not expect the content under it based on the name. Maybe something like this would be better?

```suggestion
== Mount path of Kafka log directories
```